pom.xml

```diff
@@ -10,7 +10,7 @@
   <groupId>com.pusher</groupId>
   <artifactId>pusher-http-java</artifactId>
   <name>Pusher HTTP Client</name>
-  <version>1.0.1-SNAPSHOT</version>
+  <version>1.0.2-SNAPSHOT</version>
```

Comment:
I think this should be a minor version change (consistent with our other libraries)